pkg/ccl/backupccl/restore_job.go

```diff
@@ -754,18 +754,33 @@ func getStatisticsFromBackup(
 // being restored. If the descriptorRewrites can re-write the table ID, then that
 // table is being restored.
 func remapRelevantStatistics(
-	tableStatistics []*stats.TableStatisticProto, descriptorRewrites DescRewriteMap,
+	ctx context.Context,
+	tableStatistics []*stats.TableStatisticProto,
+	descriptorRewrites DescRewriteMap,
+	tableDescs []*descpb.TableDescriptor,
 ) []*stats.TableStatisticProto {
 	relevantTableStatistics := make([]*stats.TableStatisticProto, 0, len(tableStatistics))
 
+	tableHasStatsInBackup := make(map[descpb.ID]struct{})
 	for _, stat := range tableStatistics {
+		tableHasStatsInBackup[stat.TableID] = struct{}{}
 		if tableRewrite, ok := descriptorRewrites[stat.TableID]; ok {
 			// Statistics imported only when table re-write is present.
 			stat.TableID = tableRewrite.ID
 			relevantTableStatistics = append(relevantTableStatistics, stat)
 		}
 	}
 
+	// Check if we are missing stats for any table that is being restored. This
+	// could be because we ran into an error when computing stats during the
+	// backup.
+	for _, desc := range tableDescs {
+		if _, ok := tableHasStatsInBackup[desc.GetID()]; !ok {
+			log.Warningf(ctx, "statistics for table %d not found in the backup. "+
```

Comment:
I think you're missing the table ID after this formatted string, but let's also log the table name for readability.

Reply:
done.